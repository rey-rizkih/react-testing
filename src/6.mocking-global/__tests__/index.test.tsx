import React from "react";

import { render } from "@testing-library/react";

import Galery from "..";

describe("Mocking", () => {
  test("todos list", async () => {
    const { container } = render(<Galery />);

    expect(container).toMatchSnapshot();
  });
});
