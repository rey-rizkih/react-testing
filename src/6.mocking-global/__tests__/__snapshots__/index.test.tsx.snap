// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Mocking todos list 1`] = `
<div>
  <img
    alt="image"
    data-testid="mock-next-image"
    src="https://placehold.co/600x400"
  />
</div>
`;
